tools/nxstyle.c

```diff
@@ -50,6 +50,45 @@
 
 #define LINE_SIZE    512
 
+#define FATAL(m)       message(FATAL, (m), 0, 0)
+#define FATALFL(m,s)   message(FATAL, (m), -1, -1)
+#define WARN(m, l, o)  message(WARN, (m), (l), (0))
+#define ERROR(m, l, o) message(ERROR, (m), (l), (0))
+#define INFO(m, l, o)  message(INFO, (m), (l), (0))
+
+/****************************************************************************
+ * Private types
+ ****************************************************************************/
+
+enum class_e {
+  INFO,
+  WARN,
+  ERROR,
+  FATAL
+};
+
+const char *class_text[] =
+{
+  "info",
+  "warning",
+  "error",
+  "fatal"
+};
+
+enum file_e {
+  unknown,
```

Comment:
{ belongs on a separate line.

Reply:
Use the button and you can fix
![image](https://user-images.githubusercontent.com/1945821/71326584-bd2ee580-24b1-11ea-83c6-ef83b7a2c4d1.png)

Reply:
Fix it yourself